package.json

```diff
@@ -89,7 +90,7 @@
     "jquery": "~3.4.1",
     "jquery.cookie": "~1.4.1",
     "json-loader": "^0.5.7",
-    "marked": "^0.6.3",
+    "marked": "^0.7.0",
```

Comment:
Please leave marked version as it is if you don't need it because it has a large influence.

Reply:
OK! I already fixed.